nucleus/metrics/cuboid_utils.py

```diff
@@ -101,18 +101,25 @@ def wrapper(
     return wrapper
 
 
-def process_dataitem(dataitem):
+def process_dataitem(item_list, confidence_threshold=None):
```

Comment:
This is a pretty generic name that could be more concrete. If you have the context it would be nice to re-name or add a docstring to explain what this "processing" is doing.

Reply:
I don't really have much context but I agree with you this is a terrible name... I will try to make something better